.github/workflows/lint_template.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  workflow_call:
+      inputs:
+          modulepath:
+            required: true
+            type: string
+          go-version:
+            required: true
+            type: string
+
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ inputs.go-version }}
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Install database dependencies
+        run: sudo apt-get install -y libleveldb-dev librocksdb-dev
```

Comment:
Just curious, why do we need these?

Reply:
to execute all code, including cleveldb ones. I kept rocksdb just in case we add it again in the future. One alternative is to run everything with cgo disabled.